observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -34,14 +34,25 @@ The Cluster Logging Operator manages the deployment and configuration of the col
 == Validation
 Logging includes extensive validation rules and default values to ensure a smooth and error-free configuration experience. The `ClusterLogForwarder` resource enforces validation checks on required fields, dependencies between fields, and the format of input values. Default values are provided for certain fields, reducing the need for explicit configuration in common scenarios.
 
-=== Quick Start
+== Quick Start
+
+OpenShift Logging supports two data models:
+
+* ViaQ (General Availability)
+* OTEL (Technology Preview)
+
+You can select either of these data models based on your requirements. The ViaQ data model is the default and fully supported option, while OpenTelemetry (OTEL) uses it's standards and is not fully supported. For more information, see xref:../../../observability/logging/logging-6.0/log6x-otel-data-model.adoc#log6x-otel-data-model[OTEL data model].
+
+=== Quick Start with ViaQ
```

Comment:
I suggest to add a note at the end of this section after the `ClusterLogForwarder` resource explaining that the field `lokistack.dataModel: Viaq` is the current default.